### PR TITLE
Add room topic to right panel room info

### PR DESCRIPTION
**Pull request by t3chguy**
Fixes https://github.com/element-hq/element-web/issues/26595

### CODE REVIEW

File: playwright/snapshots/invite/invite-dialog.spec.ts/invite-dialog-room-with-user-pill-linux.png
Comment (dbkr) [inline playwright/snapshots/invite/invite-dialog.spec.ts/invite-dialog-room-with-user-pill-linux.png:1]: Do we know what's appeared in the background here? Is this because of things shifting due to stable scrollbar gutter or something? It just seems unrelated to the change.

Comment (t3chguy) [reply playwright/snapshots/invite/invite-dialog.spec.ts/invite-dialog-room-with-user-pill-linux.png:1]: Yeah its the bottom right corner background under the modal, presumably due to the scrollbar shift in the right panel

Comment (dbkr) [reply playwright/snapshots/invite/invite-dialog.spec.ts/invite-dialog-room-with-user-pill-linux.png:1]: Fair enough, I guess this is what it is.

**Review by dbkr — APPROVED** [review]
